concepts/05 UI Components/Autocomplete/00 Getting Started with Autocomplete/25 Customize Item Appearance.md

```diff
@@ -0,0 +1,91 @@
+To customize appearance of all Autocomplete suggestions, define an [itemTemplate](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#itemTemplate). In React, implement the [itemRender](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#itemRender) or [itemComponent](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#itemComponent) function. To customize an individual suggestion, specify the [template](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#template) property of the item. Note that Angular and Vue use [custom templates](/Documentation/Guide/UI_Components/Common/Templates/#Custom_Templates) instead of the [template](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#template) property. In React, specify the [render](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#render) or [component](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#component) property.
+
+If your data is grouped, you may want to customize the group headings. Define a [groupTemplate](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupTemplate) for that purpose. In React, implement the [groupRender](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupRender) or [groupComponent](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupComponent) function.
```

Comment:
Likewise. `groupTemplate` is a name, not two words "group template", you cannot use "groupTemplate" instead of "group template" in a sentence as if nothing happened.

The rest of the API member names are used correctly.

```suggestion
If your data is grouped, you may want to customize the group headings. Define a [group template](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupTemplate) for that purpose. In React, implement the [groupRender](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupRender) or [groupComponent](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupComponent) function.
```